lib/python/picongpu/plugins/energy_histogram.py

```diff
@@ -111,46 +115,67 @@ def get(self, species, species_filter="all", iteration=None,
             (defined in ``particleFilters.param``)
         iteration : (unsigned) int [unitless]
             The iteration at which to read the data.
-            @TODO also allow lists here
+            A list of iterations is allowed as well.
         include_overflow : boolean, default: False
             Include overflow and underflow bins as the first/last bins.
 
         Returns
         -------
         energies : np.array of dtype float [unitless]
             count of particles in each bin
-            @todo if iteration is a list, return a dict
+            If iteration is a list, returns a dict
```

Comment:
ah, that's not true. I return a matrix with the first index being the iteration in the order as the user requested it.